lldb/scripts/clang-explicit-module-build.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
This would potentially fit better into llvm-project/lldb/packages/Python/lldbsuite/

Reply:
This is a compiler launcher type of script and requires a just built clang (because it needs clang-scan-deps next to it). I don't what is a good way to hack into lldbsuite or driven from makefile. Let me know if you have any good idea.